### ISSUE TITLE
no-deps option is not respected by omnibus recipes

### ISSUE THREAD

**Issue by brian-oconnell** [bernd/fpm-cookery#124]
```
sudo fpm-cook package --no-deps --tmp-root /tmp/fpm-temp --cache-dir /tmp/fpm-cache --pkg-dir /home/vagrant/build/pkg
===> Recipe One is an Omnibus package; looking for child recipes to build
===> Loading dependency recipe two from /home/vagrant/build/recipes/two.rb
===> Loading dependency recipe three from /home/vagrant/build/recipes/three.rb
===> Located recipe for child recipe two; starting build
===> Starting package creation for two - (redhat, rpm)
===> 
===> Verifying build_depends and depends with Puppet
===> Verifying package: ksh
===> Verifying package: compat-glibc
===> Verifying package: compat-glibc-headers
===> Verifying package: glibc-headers
===> Verifying package: glibc-devel
```

**Comment by brian-oconnell** [comment]
PR Submitted to resolve:  https://github.com/bernd/fpm-cookery/pull/125

**Comment by bernd** [comment]
Fixed in #125
